stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/ImageAttachmentContent.kt

```diff
@@ -98,7 +117,7 @@ public fun ImageAttachmentContent(
             Column(
                 modifier = Modifier
                     .weight(1f, fill = false)
-                    .fillMaxHeight(),
+                    .aspectRatio(0.5f),
```

Comment:
I'm not 100% sure what `0.5f` ratio means - so a constant would make sense here too

Reply:
Each column takes up 0.5x of the total width of the message bubble, so making it twice the size equals to a square when taking both into account-